### PR TITLE
XFF tutorial

### PR DESCRIPTION
**Pull request by triffer**
**Description**

Changes proposed in this pull request:

- Create XFF tutorial to replace the outdated blogpost

**Related issue(s)**

### CODE REVIEW

File: docs/user/tutorials/01-00-x-forwarded-for-header.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+* An enabled Istio module or an installed [Kyma Istio Operator](../../../README.md#install-kyma-istio-operator-and-istio-from-the-latest-release)
+* An enabled API Gateway module or an installed [Kyma API Gateway Operator](https://github.com/kyma-project/api-gateway#installation). Alternatively, a [custom Istio Gateway](https://kyma-project.io/#/api-gateway/user/tutorials/01-20-set-up-tls-gateway) can be used.
+
+## Steps
+
+### Configure the number of trusted proxies in the Istio CR
+
+Applications rely on reverse proxies to forward the client IP address in a request using the XFF header. However, due to 
+the variety of network topologies, configuration property numTrustedProxies must be specified with the number of trusted proxies deployed 
+in front of the Istio Gateway proxy, so that the client address can be extracted correctly.
```
Comment (nataliasitko) [inline docs/user/tutorials/01-00-x-forwarded-for-header.md:22]:
```suggestion
in front of the Istio Gateway proxy.
```

File: docs/user/tutorials/01-00-x-forwarded-for-header.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      ![Add the numTrustedProxies](./assets/01-00-num-trusted-proxies-ui.svg)
+<!-- tabs:end -->
+
+
+### Create a workload for verification
+
+1. [Create an HttpBin workload](https://kyma-project.io/#/api-gateway/user/tutorials/01-00-create-workload).
+2. Expose the HttpBin workload using a VirtualService.
+    ```bash
+    export DOMAIN_TO_EXPOSE_WORKLOADS={KYMA_DOMAIN_NAME}
```
Comment (strekm) [inline docs/user/tutorials/01-00-x-forwarded-for-header.md:47]: this one assumes that api-gateway module is there and default gateway is enabled. should we promote instead own gateway and own domain?

File: docs/user/tutorials/01-00-x-forwarded-for-header.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+To forward client attributes to destination workloads, proxies use the X-Forwarded-For (XFF) header. For more information on XFF, see 
+the [IETF’s RFC documentation](https://datatracker.ietf.org/doc/html/rfc7239) and [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for).
+
+> **NOTE:** The X-Forwarded-For header is only supported on AWS clusters.
+
+## Prerequisites
+
+* An AWS cluster
+* The Istio module enabled or [Kyma Istio Operator](../../../README.md#install-kyma-istio-operator-and-istio-from-the-latest-release) installed
+* The API Gateway module enabled or [Kyma API Gateway Operator](https://github.com/kyma-project/api-gateway#installation) installed. Alternatively, you can use [custom Istio Gateway](https://kyma-project.io/#/api-gateway/user/tutorials/01-20-set-up-tls-gateway).
```
Comment (strekm) [inline docs/user/tutorials/01-00-x-forwarded-for-header.md:14]: is it necessary to mix in api-gateway module?

File: docs/user/tutorials/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# Tutorials
+
+Browse the Istio tutorials to learn how to configure Istio to forward client information:
+
+- [Forward Client IP in X-Forwarded-For header](./01-00-x-forwarded-for-header.md)
```
Comment (strekm) [inline docs/user/tutorials/README.md:5]: @nataliasitko i'm assuming we will have more tutorials in the future

File: docs/user/tutorials/01-00-x-forwarded-for-header.md (hunk 3 above)
Comment (werdes72) [reply docs/user/tutorials/01-00-x-forwarded-for-header.md:14]: This mentions both approaches, I think it should stay like this or maybe just change the order.

File: docs/user/tutorials/01-00-x-forwarded-for-header.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+Applications rely on reverse proxies to forward the client IP address in a request using the XFF header. However, due to 
+the variety of network topologies, you must specify the configuration property **numTrustedProxies**, so that the client address can be extracted correctly. This property indicates the number of trusted proxies deployed
+in front of the Istio Gateway proxy.
+
+Add **numTrustedProxies** to the Istio custom resource (CR):
+
+<!-- tabs:start -->
+   #### **kubectl**
+  Run the following command:
+      ```bash
```
Comment (werdes72) [inline docs/user/tutorials/01-00-x-forwarded-for-header.md:29]: `bash` doesn't seem to work in a tabs view

File: docs/user/_sidebar.md [hunk 6]
```diff
@@ -18,4 +18,6 @@
   * [No Istio Sidecar](/istio/user/troubleshooting/03-30-istio-no-sidecar.md)
   * [Incompatible Istio Sidecar](/istio/user/troubleshooting/03-40-incompatible-istio-sidecar-version.md)
   * [Unintentional Istio Removal](/istio/user/troubleshooting/03-50-recovering-from-unintentional-istio-removal.md)
-  * [Not Found](/istio/user/troubleshooting/03-60-404-on-istio-gateway.md)
\ No newline at end of file
+  * [Not Found](/istio/user/troubleshooting/03-60-404-on-istio-gateway.md)
+* [Tutorials](/istio/user/tutorials/README.md)
+  *[Forward Client IP](/istio/user/tutorials/01-00-x-forwarded-for-header.md)
```
Comment (werdes72) [inline docs/user/_sidebar.md:23]:
```suggestion
  * [Forward Client IP](/istio/user/tutorials/01-00-x-forwarded-for-header.md)
```
Missing space probably

**Review by werdes72 — APPROVED** [review]

File: docs/user/tutorials/01-00-x-forwarded-for-header.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+To forward client attributes to destination workloads, proxies use the X-Forwarded-For (XFF) header. For more information on XFF, see 
+the [IETF’s RFC documentation](https://datatracker.ietf.org/doc/html/rfc7239) and [Envoy documentation](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_conn_man/headers#x-forwarded-for).
+
+> **NOTE:** The X-Forwarded-For header is only supported on AWS clusters.
+
+## Prerequisites
+
+* An AWS cluster
+* The Istio module enabled or [Kyma Istio Operator](../../../README.md#install-kyma-istio-operator-and-istio-from-the-latest-release) installed
+* An [Istio Gateway](https://kyma-project.io/#/api-gateway/user/tutorials/01-20-set-up-tls-gateway)
```
Comment (nataliasitko) [inline docs/user/tutorials/01-00-x-forwarded-for-header.md:14]:
```suggestion
* [Istio Gateway](https://kyma-project.io/#/api-gateway/user/tutorials/01-20-set-up-tls-gateway) set up
```

File: docs/user/tutorials/01-00-x-forwarded-for-header.md [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+3. Click **Edit**.
+4. Add the number of trusted proxies and click **Update**.
+   ![Add the numTrustedProxies](./assets/01-00-num-trusted-proxies-ui.svg)
+<!-- tabs:end -->
+
+
+### Create a workload for verification
+
+1. [Create a HttpBin workload](https://kyma-project.io/#/api-gateway/user/tutorials/01-00-create-workload).
+2. Expose the HttpBin workload using a VirtualService.
```
Comment (nataliasitko) [inline docs/user/tutorials/01-00-x-forwarded-for-header.md:46]: Last suggestion, could we split this step into two? Like this:
2. Export the following values as environment variables.
 ```bash
    export DOMAIN_TO_EXPOSE_WORKLOADS={GATEWAY_DOMAIN}
    export GATEWAY={GATEWAY_NAMESPACE}/GATEWAY_NAME}
 ```
3. Expose the HttpBin workload using a VirtualService.
 ```bash
    cat <<EOF | kubectl apply -f -
    ...
 ```

**Review by nataliasitko — APPROVED** [review]

**Pull request merged**